src/react-components/profile-entry-panel.js

```diff
@@ -134,12 +134,12 @@ export default class ProfileEntryPanel extends Component {
         this.props.mediaSearchStore.sourceNavigateWithNoNav("avatars", "use");
       },
       onSubmit: this.saveStateAndFinish,
-      showBackButton: this.props.showBackButton,
       onClose: this.props.onClose,
       onBack: this.props.onBack
     };
 
     if (this.props.containerType === "sidebar") {
+      avatarSettingsProps.showBackButton = this.props.showBackButton;
       return <AvatarSettingsSidebar {...avatarSettingsProps} />;
```

Comment:
Rather than mutating the object in a render method, you can just pass in the prop:
`<AvatarSettingsSidebar {...avatarSettingsProps} showBackButton={this.props.showBackButton} />`

Generally we should avoid mutation in render methods, even though in this case it seems acceptable.